core/driver/sources/src/main/scala/org/virtuslab/ideprobe/ide/intellij/InstalledIntelliJ.scala

```diff
@@ -128,3 +130,24 @@ final class InstalledIntelliJ(val root: Path, probePaths: IdeProbePaths, config:
     builder.start()
   }
 }
+
+final class LocalIntelliJ(
+    val root: Path,
+    probePaths: IdeProbePaths,
+    config: DriverConfig,
+    private val pluginsBackup: Path
+) extends InstalledIntelliJ(root, probePaths, config) {
+  override def cleanup(): Unit = {
+    val pluginsDir = root.resolve("plugins")
+    pluginsDir.delete()
+    Files.move(pluginsBackup, pluginsDir, StandardCopyOption.REPLACE_EXISTING)
```

Comment:
I think we have a method for that in pathextension, if not, it's best to add one

Reply:
Done.